### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by Kerruba**
Removes EntandoProperties class and refactor code to get values direc…tly from application.yml file using @Value annotation

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

File: src/main/java/org/entando/plugin/avatar/config/ConfigServiceConfiguration.java [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
 
 import org.entando.config.ConfigService;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class ConfigServiceConfiguration {
 
+    @Value("security.oauth.client.client-id")
```
Comment (ampie) [inline src/main/java/org/entando/plugin/avatar/config/ConfigServiceConfiguration.java]: Is there a reason we are not using propertyResolver expressions (e.g. ${asdf.asfd.sfda}) here?

Comment (Kerruba) [reply src/main/java/org/entando/plugin/avatar/config/ConfigServiceConfiguration.java]: I've fixed those, thanks for spotting it

**Comment by entando-service** [comment]
All is well

**Pull request merged**